@types/slate-react/index.d.ts

```diff
@@ -0,0 +1,68 @@
+declare module 'slate-react' {
```

Comment:
I'm a bit lost here 😅 I'm not sure if I understand.
We are using `slate-react` [here](https://github.com/commercetools/ui-kit/blob/8ac71c4ef1ea27566a82fd3cafbca3ad7ddda871/packages/components/inputs/rich-text-input/src/rich-text-input.tsx#L2) and that was one of the ways of getting type declarations. Considering that we have recently typed [`TEditor`](https://github.com/commercetools/ui-kit/blob/main/packages/components/inputs/rich-text-utils/src/editor.types.ts) on our own that was the way I thought of. Of course only as a temporary solution easily removable once we migrate to the latest (TS-based) version of `slate-`. The other option was using `// @ts-ignore` but I assumed that was out of the question 😅
I am wondering if you don't approve of that at all? In that case I would need a suggestion what would be the appropriate way

Reply:
FYI: @kark and me took a deeper look at this and decided to remove the type declarations and have a wrapper (typed) component for the `Editor` component (`slate-react`).

This should be all we need, as the `TEditor` object is not the same as the component and it can stay in the utils package.